go/flags/endtoend/mysqlctld.txt

```diff
@@ -110,6 +110,7 @@ Flags:
       --replication_connect_retry duration                               how long to wait in between replica reconnect attempts. Only precise to the second. (default 10s)
       --security_policy string                                           the name of a registered security policy to use for controlling access to URLs - empty means allow all for anyone (built-in policies: deny-all, read-only)
       --service_map strings                                              comma separated list of services to enable (or disable if prefixed with '-') Example: grpc-queryservice
+      --shutdown-wait-time duration                                      How long to wait for mysqld shutdown (default 5m0s)
```

Comment:
Does this have to be a new flag? Can you not just pass a higher value for wait_time?